### PR TITLE
Add total ADU to blocklist add page

### LINKED ISSUE #16674: Show the total amount of affected users on the blocklist page
### Describe the problem and steps to reproduce it:

When blocking several add-ons in bulk, there is a nice list of add-ons involved in the block, along with usage numbers for each.

### What happened?

I can't easily see how many affected users there are in total.

### What did you expect to happen?

That page could also show the total number of users.

### Anything else we should know?
(Please include a link to the page, screenshots and any relevant files.)

### PR DESCRIPTION
**Pull request by wagnerand**
Fixes #16674.

### CODE REVIEW

File: src/olympia/blocklist/tests/test_admin.py [hunk 1]
```diff
@@ -490,6 +490,8 @@ def _test_add_multiple_submit(self, addon_adu):
         content = response.content.decode('utf-8')
         # meta data for new blocks and existing ones needing update:
         assert 'Add-on GUIDs (one per line)' not in content
+        total_adu = addon_adu + addon_adu - 1
```
Comment (willdurand) [inline src/olympia/blocklist/tests/test_admin.py:493]: mm, where does that come from?

Comment (wagnerand) [reply src/olympia/blocklist/tests/test_admin.py:493]: That's being defined a few lines above, line 458 and 471

Comment (willdurand) [reply src/olympia/blocklist/tests/test_admin.py:493]: I suggest we introduce variables for these then, like `self.new_addon_adu`.

Comment (wagnerand) [reply src/olympia/blocklist/tests/test_admin.py]: d40eb14dc3

**Review by willdurand — APPROVED** [review]

**Pull request merged**